src/SQLProvider/Utils.fs

```diff
@@ -384,8 +431,8 @@ module Sql =
         }
 
     let executeSql createCommand sql (con:IDbConnection) =        
-        use com : IDbCommand = createCommand sql con   
-        com.ExecuteReader()    
+        let com : IDbCommand = createCommand sql con   
```

Comment:
What is the point of new DataReaderWithCommand-type?
How much longer will the command live in the memory when instead of disposing it, it is captured to this new type?